recipes/hoomd-dlext/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set processor = "cpu" %}
+{% set name = "hoomd-dlext" %}
+{% set version = "0.2.1" %}
+{% set build = 0 %}
+{% set sha256 = "547316c95608d7ee1ca11ace53ccf22dddc88206fa0ac0613020f48bf6081269" %}
+{% set processor = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [linux64]
+{% set processor = "cpu" %}  # [not linux64]
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SSAGESLabs/hoomd-dlext/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  # Upstream package does not support Windows, Python < 3.6, or PyPy
+  skip: true  # [win or py<36 or python_impl == "pypy"]
+  string: {{ processor }}_py{{ py }}h{{ PKG_HASH }}_{{ build }}
+  ignore_run_exports: tbb
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux64 and cuda_compiler_version != "None"]
+    - cmake >=3.16
+    - make
+
+  host:
+    - python
+    - pybind11
+    - dlpack >=0.5
+    - hoomd <3
+    - tbb-devel
+
+  run:
+    - {{ pin_compatible('hoomd', max_pin='2.x') }}
```

```suggestion
    - {{ pin_compatible('hoomd', max_pin='x.x') }}
```
Pin already restricted to <3.0 from host spec.